clipboard/clipboard.d.ts

```diff
@@ -3,8 +3,8 @@
 // Definitions by: Andrei Kurosh <https://github.com/impworks>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare class Clipboard {
-    constructor(selector: (string | Element | NodeListOf<Element>), options?: ClipboardOptions);
+interface ClipboardStatic {
+    new(selector: (string | Element | NodeListOf<Element>), options?: ClipboardOptions): ClipboardStatic;
```

Comment:
so `new ClipboardStatic (..)` returns another `ClipboardStatic`, that does not sound right.